c2corg_api/search/mappings/book_mapping.py

```diff
@@ -0,0 +1,45 @@
+from c2corg_api.models.book import BOOK_TYPE, Book
+from c2corg_api.search.mapping import SearchDocument, BaseMeta
+from c2corg_api.search.mapping_types import QueryableMixin, QEnumArray, QEnum
+
+
+class SearchBook(SearchDocument):
+    class Meta(BaseMeta):
+        doc_type = BOOK_TYPE
+
+    activities = QEnumArray(
+        'act', model_field=Book.activities)
+    book_types = QEnumArray(
+        'btyp', model_field=Book.book_types)
+
+    isbn = QEnum(
+        'isbn', model_field=Book.isbn)
+    author = QEnum(
+        'auth', model_field=Book.author)
+    editor = QEnum(
```

Comment:
I am not sure about the type for `author` and `editor`. Should these two fields be searchable? In the full-text-search?